src/_tests_/addRemove.test.js

```diff
@@ -0,0 +1,18 @@
+import { addTask, removeTask } from '../_mock_/addTask.js';
+
+describe('Todo list functionality', () => {
+  let tasks = [];
+  const todoStr = 'test';
+  // addTodo test
+  test('addTask', () => {
+    addTask(tasks, todoStr);
+    expect(tasks).toHaveLength(1);
+    expect(tasks[0].description).toBe('test');
+  });
+
+  test('removeTask', () => {
+    const todo = tasks[0];
+    tasks = removeTask(todo, tasks);
+    expect(tasks).toHaveLength(0);
+  });
+});
```

Comment:
- [ ] Good job setting up the tests 👍 however these tests are not testing the actual code, they are just testing the mock functions, please make sure that your tests are actually testing your code.